packages/engine/Source/Scene/Cesium3DTileset.js

```diff
@@ -1808,6 +1812,42 @@ Object.defineProperties(Cesium3DTileset.prototype, {
     },
   },
 
+  /**
+   * A Property specifying if the height is relative to terrain, 3D Tiles, or both.
```

Comment:
```suggestion
   * Specifies if the height is relative to terrain, 3D Tiles, or both.
```

A `Property` is a specific type that is used with Entities, but is not used here.